install_scripts/carma-messenger_setup.bash

```diff
@@ -33,14 +28,14 @@ sudo chmod 775 -R /opt/carma-messenger/
 sudo chmod 775 /opt/carma-messenger/Desktop /opt/carma-messenger/Pictures
 
 
-curl -o /opt/carma-messenger/Desktop/CARMA_Messenger.desktop -L https://raw.githubusercontent.com/usdot-fhwa-stol/carma-messenger/feature/login_logout/install_scripts/Desktop/CARMA_Messenger.desktop 
+curl -o /opt/carma-messenger/Desktop/CARMA_Messenger.desktop -L https://raw.githubusercontent.com/usdot-fhwa-stol/carma-messenger/tree/develop/install_scripts/Desktop/CARMA_Messenger.desktop 
```

Comment:
Refer to the URL path used in https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/engineering_tools/opt_carma_setup.bash : 
Seems this should say: https://raw.githubusercontent.com/usdot-fhwa-stol/carma-messenger/develop/install_scripts/Desktop/CARMA_Messenger.desktop, right?